无法读取@全体的消息

系统Ubuntu2404，Napcat
消息为
```
03-07 22:11:01 [info] *** | 接收 <- 群聊 [***(10***)] [***(36***)] @全体成员  搞了个新家伙 [表情 /汪汪]
```
报错为：
```
03-07 22:11:01 [ERROR] nonebot | Running Matcher(type='message', module=src.plugins.chat, lineno=49) failed.
Traceback (most recent call last):
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/bot.py", line 83, in <module>
    nonebot.run()
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/__init__.py", line 337, in run
    get_driver().run(*args, **kwargs)
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/drivers/fastapi.py", line 187, in run
    uvicorn.run(
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/uvicorn/main.py", line 579, in run
    server.run()
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/lib/python3.12/asyncio/runners.py", line 194, in run
    return runner.run(main)
  File "/usr/lib/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/utils.py", line 254, in run_coro_with_shield
    return await coro
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/message.py", line 506, in check_and_run_matcher
    await _run_matcher(
> File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/message.py", line 458, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/internal/matcher/matcher.py", line 926, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/internal/matcher/matcher.py", line 863, in simple_run
    await handler(
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/venv/lib/python3.12/site-packages/nonebot/dependencies/__init__.py", line 113, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/src/plugins/chat/__init__.py", line 88, in _
    await chat_bot.handle_message(event, bot)
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/src/plugins/chat/bot.py", line 52, in handle_message
    message = Message(
  File "<string>", line 19, in __init__
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/src/plugins/chat/message.py", line 67, in __post_init__
    self.message_segments = self.parse_message_segments(str(self.raw_message))
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/src/plugins/chat/message.py", line 139, in parse_message_segments
    message_obj = cq_code_tool.cq_from_dict_to_class(_code_item,reply = self.reply_message)
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/src/plugins/chat/cq_code.py", line 365, in cq_from_dict_to_class
    instance.translate()
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/src/plugins/chat/cq_code.py", line 77, in translate
    user_nickname = get_user_nickname(self.params.get('qq', ''))
  File "/home/lxmalpha/Bot-Project/Nonebot/MaiMBot-0.5.8-alpha/src/plugins/chat/utils_user.py", line 5, in get_user_nickname
    if int(user_id) == int(global_config.BOT_QQ):
ValueError: invalid literal for int() with base 10: 'all'
```

--- a/src/plugins/chat/cq_code.py
+++ b/src/plugins/chat/cq_code.py
@@ -86,9 +86,12 @@ class CQCode:
             else:
                 self.translated_segments = Seg(type="text", data="[图片]")
         elif self.type == "at":
-            user_nickname = get_user_nickname(self.params.get("qq", ""))
-            self.translated_segments = Seg(
-                type="text", data=f"[@{user_nickname or '某人'}]"
+            if self.params.get("qq") == "all":
+                self.translated_segments = Seg(type="text", data="@[全体成员]")
+            else:  
+                user_nickname = get_user_nickname(self.params.get("qq", ""))
+                self.translated_segments = Seg(
+                    type="text", data=f"[@{user_nickname or '某人'}]"
             )
         elif self.type == "reply":
             reply_segments = self.translate_reply()

